.github/actions/install-protoc/action.yml

```diff
@@ -0,0 +1,28 @@
+name: Install Protoc
+description: Install Protocol Buffers compiler (protoc)
+
+inputs:
+  version:
+    description: "Protoc version to install"
+    required: false
+    default: "30.2"
+  arch:
+    description: "Architecture for the protoc binary"
+    required: false
+    default: "linux-x86_64"
+
+runs:
+  using: "composite"
+  steps:
+    - name: Install protoc for the gRPC extension
```

Comment:
perhaps use this action instead? https://github.com/marketplace/actions/setup-protoc

Reply:
done